webapp/src/components/progress_bars/TrainingBar.vue

```diff
@@ -1,50 +1,42 @@
 <template>
-  <div class="mb-8 md:mb-16 space-y-4 md:space-y-8">
+  <div class="mb-8 space-y-4 md:space-y-8">
     <div
-      v-if="scheme !== undefined && displayTitle"
+      v-if="taskTitle !== undefined && displayTitle"
       class="flex flex-wrap font-disco text-3xl justify-center"
     >
-      <span class="text-disco-blue" style="text-transform: capitalize">{{ scheme }}</span>
-      <span class="text-disco-cyan">&nbsp;Learning</span>
+      <span class="text-disco-blue">{{ taskTitle }}</span>
+      <!-- <span class="text-disco-blue" style="text-transform: capitalize">{{ scheme }}</span>
+      <span class="text-disco-cyan">&nbsp;Learning</span> -->
     </div>
     <div
       v-else
       class="flex flex-wrap text-3xl text-slate-600 justify-center"
     >
-      <DISCOllaborative />
+      <DISCOllaboratives />
     </div>
     <div class="hidden md:inline-block w-full py-6">
       <div class="flex">
         <!-- Step 1 -->
         <ProgressIcon
           class="w-1/5"
           :active="true"
-          :lined="false"
+          :current-step="trainingStore.step == 0"
+          :has-left-line="false"
           @click="toStep(0)"
         >
           <template #text>
             Task Selection
           </template>
           <template #icon>
-            <svg
-              class="w-full fill-current"
-              xmlns="http://www.w3.org/2000/svg"
-              viewBox="0 0 24 24"
-              width="24"
-              height="24"
-            >
-              <path
-                class="heroicon-ui"
-                d="M5 3h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5c0-1.1.9-2 2-2zm14 8V5H5v6h14zm0 2H5v6h14v-6zM8 9a1 1 0 1 1 0-2 1 1 0 0 1 0 2zm0 8a1 1 0 1 1 0-2 1 1 0 0 1 0 2z"
-              />
-            </svg>
+            <TasksIcon  customClass="w-full w-7 h-7" view-box="-4 -4 24 24"/>
```

Comment:
(nitpicking: double space)
is it not working as wanted to simply set the class on the component?

```suggestion
            <TasksIcon class="w-full w-7 h-7" view-box="-4 -4 24 24"/>
```

Reply:
Not really when I tried and it seems to be a common issue: https://dev.to/saifobeidat/override-tailwind-classes-of-a-reusable-vue-component-without-using-props-660